### DISCUSSION TITLE
Using the InvenTree "Upload File for Purchase Order" Function

### DISCUSSION THREAD

**Discussion by palagap** [inventree/InvenTree#3033] [Q&A] [answered]
Hi there!

I'm currently following these instructions to setup a PO in InvenTree: https://inventree.readthedocs.io/en/latest/companies/po/. 

I've been using the demo account listed here (with the admin account): https://inventree.readthedocs.io/en/latest/demo/. 

Everything in the instructions seems to work well, except for the upload of an exported purchase order. I’ve been testing the exported POs from DigiKey in both .xlsx and .csv formats, but for some reason the system can’t process them. Manually entering line items works, though. 

Specifically, I get stuck on the second page of the "Upload File for Purchase Order" section. It appears that only six fields are available as headers for the file, and even when I modify a standard DigiKey PO .csv to match these fields, I get the message "Fields matching failed". Any ideas how how to address this? Screenshots included below. Thanks.

<img width="1684" alt="Screen Shot 2022-05-18 at 4 29 28 PM" src="https://user-images.githubusercontent.com/40808444/169172154-aaa18221-cc0b-4e26-b8e5-c377e7dff545.png">
<img width="1697" alt="Screen Shot 2022-05-18 at 4 29 43 PM" src="https://user-images.githubusercontent.com/40808444/169172157-366ec26a-d73c-4e7f-bab7-44a061e663dc.png">
<img width="1669" alt="Screen Shot 2022-05-18 at 4 29 57 PM" src="https://user-images.githubusercontent.com/40808444/169172159-07b91ce9-1a24-4608-b070-0ea35cf5b3f7.png">
<img width="1161" alt="Screen Shot 2022-05-18 at 4 30 05 PM" src="https://user-images.githubusercontent.com/40808444/169172160-1f668eab-3316-4a13-948a-0441f1e85f33.png">
<img width="1668" alt="Screen Shot 2022-05-18 at 4 30 21 PM" src="https://user-images.githubusercontent.com/40808444/169172161-cc06d75a-9683-4f8f-b239-622ed5ab1a15.png">

**Comment by lukasbrchl** [comment]
Hi everyone, we have the same issue. I have tried multiple formats, and multiple fields with dummy CSVs, but was unable to go through this step. 

@SchrodingersGat, could you please tell us, how we should go through the import form?

**Comment by eeintech** [comment, marked as answer]
@palagap @lukasbrchl I just saw this, I will check it out soon to see if anything broken (shouldn't be). Which InvenTree version are you using?

**Reply by lukasbrchl** [reply]
The latest stable one - 0.7.0. The functionality was working for us before the form overhaul (before 0.7.0). Let me know if you need further information from us.

![image](https://user-images.githubusercontent.com/18639856/170828404-85a81bbf-e715-40f7-a5d3-db88b822388d.png)

**Reply by eeintech** [reply]
@lukasbrchl Confirmation the import tool has issues now, see https://github.com/inventree/InvenTree/issues/3098

**Reply by lukasbrchl** [reply]
Thanks for confirming! Have a great day
